push/forms.py

```diff
@@ -7,6 +7,10 @@ class PushAppForm(ModelForm):
     class Meta:
         model = PushApplication
         fields = ['name', 'vapid_key']
+        widgets = {
+            'name': TextInput(attrs={'required': True}),
+            'vapid_key': TextInput(attrs={'required': True})
+        }
```

Comment:
This makes django add `required` attributes on the `input type="text"` elements for HTML(5) client-side validation.